sphinx_automodapi/tests/test_automodapi.py

```diff
@@ -1,52 +1,39 @@
 # -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-import pytest
+import sys
+from copy import copy
 
-from ..utils import iteritems
-from . import cython_testpackage
+import pytest
+from docutils.parsers.rst import directives, roles
 
-pytest.importorskip('sphinx')  # skips these tests if sphinx not present
+from . import cython_testpackage  # noqa
+from .helpers import run_sphinx_in_tmpdir
 
+if sys.version_info[0] == 2:
+    from io import open as io_open
```

Comment:
do we still need to distinguish and test with python2 `open()` or can just assume everything is `io.open`?

Reply:
I had to use ``io.open`` for this test because of the unicode, but I ran into issues if I tried to use it for all tests (and I wanted to use this in a way that it wouldn't be a pain to remove when we drop Python 2 support).